### DISCUSSION TITLE
请问为什么在转换为coco格式的时候，必须将bbox框归一化到200尺寸？

### DISCUSSION THREAD

**Discussion by Jeozhao** [open-mmlab/mmpose#3092] [General]
https://github.com/open-mmlab/mmpose/blob/b9c1b54fbe7f9bddd90206946b239dc440b0cf50/tools/dataset_converters/wflw2coco.py#L48
请问在将数据转换为coco格式的时候，为什么需要将人脸的框尺寸归一化到200大小。这有什么特别的用意吗？

**Comment by Sologala** [comment]
@Jeozhao 建议看看Dark 的论文。
